website-components-playground/package.json

```diff
@@ -10,8 +10,8 @@
     "serve": "gatsby serve"
   },
   "dependencies": {
-    "@commercetools-frontend/application-components": "15.2.1",
-    "@commercetools-frontend/i18n": "15.4.1",
+    "@commercetools-frontend/application-components": "15.4.2",
```

Comment:
```suggestion
    "@commercetools-frontend/application-components": "15.5.0",
```

Reply:
Good catch. I just went with latest.

Reply:
Yeah, we can actually promote the packages to latest. Will do that now